### PR TITLE
🇪🇪 ET Translation patch

### PR DESCRIPTION
**Pull request by exsilium**
Dear Maintainer, thank you for providing this piece of gaming history in an approachable format. I'm submitting an Estonian translation patch in the hopes that it will be accepted to the main branch.

Contents:
- First iteration of Estonian translations 🇪🇪 
- Added "Eesti" to Language options as well as included the strings to the Asset file.
- Some playtesting has been completed already to accommodate and change longer strings to shorter ones to ensure proper UI renderings without overlap.

Waiting for your feedback and hoping this patch gets accepted. A small boy is eager to cleanse Sanctuary from the evil devilish forces 🎸 

Cheers! 🍻 

d1-et-1
d1-et-2

### CODE REVIEW

File: Source/options.cpp [hunk 1]
```diff
@@ -1221,6 +1221,7 @@ void OptionEntryLanguageCode::CheckLanguagesAreInitialized() const
 	languages.emplace_back("el", "Ελληνικά");
 	languages.emplace_back("en", "English");
 	languages.emplace_back("es", "Español");
+    languages.emplace_back("et", "Eesti");
```
Comment (glebm) [inline Source/options.cpp:1224]: This should be indented with a tab rather than 4 spaces.

Comment (exsilium) [reply Source/options.cpp]: Fixed! 🚀

**Review by StephenCWills — APPROVED** [review]
Looks great.

**Comment by AJenbo** [comment]
Nice work

**Pull request merged**